src/xmipp/libraries/reconstruction/resolution_fso.cpp

```diff
@@ -25,1334 +25,1078 @@
+		MAT_ELEM(angles, 0, 58) = 180.000000;	 MAT_ELEM(angles, 1, 58) = -26.565058;
+		MAT_ELEM(angles, 0, 59) = 167.188979;	 MAT_ELEM(angles, 1, 59) = -42.234673;
+		MAT_ELEM(angles, 0, 60) = 180.000000;	 MAT_ELEM(angles, 1, 60) = -58.282544;
+		MAT_ELEM(angles, 0, 61) = 161.533003;	 MAT_ELEM(angles, 1, 61) = -59.620797;
+		MAT_ELEM(angles, 0, 62) = 171.132791;	 MAT_ELEM(angles, 1, 62) = -75.219088;
+		MAT_ELEM(angles, 0, 63) = 108.000000;	 MAT_ELEM(angles, 1, 63) = -26.565058;
+		MAT_ELEM(angles, 0, 64) = 120.811021;	 MAT_ELEM(angles, 1, 64) = -42.234673;
+		MAT_ELEM(angles, 0, 65) = 95.188979;	 MAT_ELEM(angles, 1, 65) = -42.234673;
+		MAT_ELEM(angles, 0, 66) = 126.466996;	 MAT_ELEM(angles, 1, 66) = -59.620797;
+		MAT_ELEM(angles, 0, 67) = 108.000000;	 MAT_ELEM(angles, 1, 67) = -58.282544;
+		MAT_ELEM(angles, 0, 68) = 89.533003;	 MAT_ELEM(angles, 1, 68) = -59.620797;
+		MAT_ELEM(angles, 0, 69) = 108.000000;	 MAT_ELEM(angles, 1, 69) = 90.000000;
+		MAT_ELEM(angles, 0, 70) = 116.867209;	 MAT_ELEM(angles, 1, 70) = -75.219088;
+		MAT_ELEM(angles, 0, 71) = 99.132791;	 MAT_ELEM(angles, 1, 71) = -75.219088;
+		MAT_ELEM(angles, 0, 72) = 36.000000;	 MAT_ELEM(angles, 1, 72) = -26.565058;
+		MAT_ELEM(angles, 0, 73) = 48.811021;	 MAT_ELEM(angles, 1, 73) = -42.234673;
+		MAT_ELEM(angles, 0, 74) = 23.188979;	 MAT_ELEM(angles, 1, 74) = -42.234673;
+		MAT_ELEM(angles, 0, 75) = 54.466996;	 MAT_ELEM(angles, 1, 75) = -59.620797;
+		MAT_ELEM(angles, 0, 76) = 36.000000;	 MAT_ELEM(angles, 1, 76) = -58.282544;
+		MAT_ELEM(angles, 0, 77) = 17.533003;	 MAT_ELEM(angles, 1, 77) = -59.620797;
+		MAT_ELEM(angles, 0, 78) = 36.000000;	 MAT_ELEM(angles, 1, 78) = 90.000000;
+		MAT_ELEM(angles, 0, 79) = 44.867209;	 MAT_ELEM(angles, 1, 79) = -75.219088;
+		MAT_ELEM(angles, 0, 80) = 27.132791;	 MAT_ELEM(angles, 1, 80) = -75.219088;
+	}
 
-					if (cosine>=cosAngle)
-						{
-						if (f>maxFreq)
-						{
-							++n;
-							continue;
-						}
-						int idx = (int) round(f * m1sizeX);
-						cosine = exp( -((cosine -1)*(cosine -1))*aux );
-						DIRECT_MULTIDIM_ELEM(threeD_FSC, n) += cosine*dAi(fsc, idx);
-						DIRECT_MULTIDIM_ELEM(counterMap, n) += cosine;
-						}
-					++n;
-				}
-			}
-		}
-    }
+	angles *= PI/180;
+}
 
-    //TODO: Merge with Simple
-    void anistropyParameter(const MultidimArray<double> FSC,
-    		MultidimArray<double> &directionAnisotropy, size_t dirnumber,
-			MultidimArray<double> &aniParam, double thrs)
-    {
-    	double N = 0;
-		for (size_t k = 0; k<aniParam.nzyxdim; k++)
+
+void ProgFSO::anistropyParameter(const MultidimArray<double> FSC,
+		MultidimArray<double> &directionAnisotropy, size_t dirnumber,
+		MultidimArray<double> &	aniParam, double thrs)
+{
+	double N = 0;
+	for (size_t k = 0; k<aniParam.nzyxdim; k++)
+	{
+		if (DIRECT_MULTIDIM_ELEM(FSC, k) >= thrs)
 		{
-			if (DIRECT_MULTIDIM_ELEM(FSC, k) >= thrs)
-			{
-				DIRECT_MULTIDIM_ELEM(aniParam, k) += 1.0;
-				N++;
-			}
+			DIRECT_MULTIDIM_ELEM(aniParam, k) += 1.0;
```

Comment:
I would prefer if you init aniParam to zero in this method

Reply:
it was initialized before, not in the method, because it changes in the loop